R/commonMachineLearningClassification.R

```diff
@@ -753,7 +817,9 @@
   classProportionsTable$position <- position
   classProportionsTable$dependOn(options = c("classProportionsTable", "noOfNearestNeighbours", "trainingDataManual", "distanceParameterManual", "weights", "scaleEqualSD", "modelOpt",
```

Comment:
If I look atlines 748-750 and here (and later) there is some duplication, can't we do:
```r
classProportionsTable$dependOn(optionsFromObject = validationMeasures)
jaspResults[["classColumn"]]$dependOn(optionsFromObject = validationMeasures)
```
?

Reply:
Did you look at this Koen?